src/languages/rust.js

```diff
@@ -318,6 +318,12 @@ export default function(hljs) {
         className: "punctuation",
         begin: '->'
       },
+      {
+        className: 'unsafe_extern',
```

Comment:
```suggestion
        scope: 'unsafe_extern',
```

Reply:
Wait, this isn't one of our named scopes... this value must be one of the scopes from our documentation.  We'll also need a markup test to show this works as intended.